README.md

```diff
@@ -28,92 +37,9 @@ Through a grant provided by [Jetstream](https://jetstream-cloud.org/), Unidata i
-* [awips2-foss](https://github.com/Unidata/awips2-foss)
-* [awips2-ncep](https://github.com/Unidata/awips2-ncep)
-* [awips2-rpm](https://github.com/Unidata/awips2-rpm)
-
-
-# Setting up the AWIPS Development Environment (ADE)
-
-Instructions on how to deploy CAVE from Eclipse.
-
-1. Change `/etc/yum.repos.d/awips2.repo` to 
-
-        [awips2repo]
-        name=AWIPS II Repository
-        baseurl=https://www.unidata.ucar.edu/repos/yum/el6-dev/
-        enabled=1
-        protect=0
-        gpgcheck=0
-        proxy=_none_
-        
-2. `yum clean all && yum groupinstall awips2-ade`
-
-    This will install Eclipse (4.6.1), Java (1.8), Ant, Maven, Python 2.7 and its modules (Numpy, Shapely, etc.) 
-
-3. `git clone https://github.com/Unidata/awips2.git`
-
-    The full list of repositories required:
-    
-        git clone https://github.com/Unidata/awips2.git
-        git clone https://github.com/Unidata/awips2-core.git
-        git clone https://github.com/Unidata/awips2-core-foss.git
-        git clone https://github.com/Unidata/awips2-foss.git
-        git clone https://github.com/Unidata/awips2-ncep.git
-        git clone https://github.com/Unidata/awips2-rpm.git
-
-    Optional repositories:
-
-        git clone https://github.com/Unidata/awips2-nws.git
-        git clone https://github.com/Unidata/awips2-gsd.git
-        git clone https://github.com/Unidata/awips2-drawing.git
-
-4. Run `/awips2/eclipse/eclipse.sh`
-
-    * Preferences > Java 
-        
-        Set to **/awips2/java**
-    
-    * Preferences > PyDev > Python Interpreter
-    
-        Set to **/awips2/python/bin/python** (should be resolved by Auto-Config)
-
-    * File > Import > General > Existing Projects Into Workspace
-    
-        Import all of the git cloned project folders **EXCEPT** for the main (first) **github.com/Unidata/awips2.git** directory (which should be **~/awips2**).
-         
-        You'll want to import **~/awips2** in three parts to ensure a clean and error-free Eclipse build:
-        
-        1. Import **awips2/cave** > Select All Projects > Finish
-        2. Import **awips2/edexOsgi** > Select All Projects > Finish
-            
-        Now import all other repositories fully: 
-        
-        Select **awips2-core**, **awips2-core-foss**, **awips2-foss**, **awips2-ncep**, etc. > Select All Projects > Finish 
-
-    * Project > Clean
-    
-        Run a clean build and ensure no errors are reported.  
-    
-    
-5. Run **com.raytheon.viz.product.awips/developer.product**
-
-    Double-click the **developer.product** file to open the Product View in Eclipse.  Select **Overview** > **Synchronize** and then right-click the file in the left-side package explorer:
-    
-    Select **Run As** > **Eclipse Application** to launch CAVE in the development environment. 
-    
-    Select **Debug** > **Eclipse Application** to launch CAVE in in debug mode. 
+	* [(click to subscribe)](mailto:awips2-users-join@unidata.ucar.edu)
```

Comment:
We don't want to include links to the other repos now? (I know the previous list was incomplete...)

Reply:
A lot of the information was outdated about the ADE and repos so I included a link to the AWIPS Development which has all of our updated information (line 41)

* [AWIPS Development Environment (ADE)](http://unidata.github.io/awips2/dev/awips-development-environment)

But if you feel strongly that I should link all of the repos I can add that back in